### DISCUSSION TITLE
"briefcase create android" command displays "Briefcase configuration error: configuration file not found"

### DISCUSSION THREAD

**Discussion by guaiguaicai** [beeware/briefcase#609] [Q&A]
**Describe the bug**

After installing beeware, it is normal to use the briefcase command to create, compile and run the App under windows or android system.

From the next day, the originally created App can be modified, compiled and run normally. The "briefcase new" and "briefcase dev" commands can also be used normally.

The "briefcase create android" command displays "Briefcase configuration error: configuration file not found".
Both of my computers have this situation. What is the reason? What should I do? Thanks!

**To Reproduce**
1. briefcase create android
2. See error   “Briefcase configuration error: configuration file not found"

**Expected behavior**
Create App for Android

**Screenshots**
(beeware)  D:\Python_Project\beeware_test
$ d:\Python\Python38\envs\beeware\Scripts\briefcase.exe create android
Briefcase configuration error: configuration file not found

**Environment:**
 - Operating System: windows 10
 - Python version: 3.8
 - Software versions: 
   - Briefcase: 0.3.5
   - Toga: 0.3.0.dev28
   - ...

**Additional context**

**Comment by freakboy3742** [comment]
The "Configuration file not found" error occurs when you run briefcase in a folder other than the one that contains `pyproject.toml`. If you're in the right directory when you run briefcase, you shouldn't get this error.

**Comment by guaiguaicai** [comment]
It has been resolved, first briefcase new, enter the app directory, and then briefcase create andriod, it will do, thank you!
